src/test/TEST_ij/smoother.jobs

```diff
@@ -84,8 +84,8 @@ mpirun -np 4  ./ij -rhsrand -solver 1 -rlx 16 -cheby_variant 1 -n 20 20 20 \
 mpirun -np 4  ./ij -solver 3 -rlx 16 -cheby_eig_est 0 -n 40 40 20 \
 -P 2 2 1 -difconv -a 10 10 10  > smoother.out.18
 
-mpirun -np 4  ./ij -rhsrand -solver 1 -rlx 16 -rotate -alpha 60 -eps 0.1 -cheby_fraction 0.2 -n 200 200 \
--P 2 2  > smoother.out.19
+mpirun -np 4  ./ij -rhsrand -solver 1 -rlx 16 -rotate -alpha 60 -eps 0.1 -cheby_fraction 0.2 -n 200 200 1 \
+-P 2 2 1 > smoother.out.19
 
```

Comment:
Note that here the '1's are actually not needed, since this is a 2D problem, but they won't hurt either.

Reply:
The reason I added them is because the argument parsing script reads three variables https://github.com/hypre-space/hypre/blob/9117fc20f2c4ed88700cd6cb1f5da2463b63c0fa/src/test/ij.c#L7785, so if it's the last argument, `argv[argc]` will be read (technically out of bounds of argv) and attempted to be converted to an int, which will be undefined (although I suspect unused). I agree that I don't think nz/R will be used in the 2D problem, so the different value won't matter.

Reply:
> The reason I added them is because the argument parsing script reads three variables
> 
> https://github.com/hypre-space/hypre/blob/9117fc20f2c4ed88700cd6cb1f5da2463b63c0fa/src/test/ij.c#L7785
> 
> , so if it's the last argument, `argv[argc]` will be read (technically out of bounds of argv) and attempted to be converted to an int, which will be undefined (although I suspect unused). I agree that I don't think nz/R will be used in the 2D problem, so the different value won't matter.

It's good to fix. It's not fun to have `argv` out-of-range. Thanks!